backend/app/adapter/routing/route.go

```diff
@@ -34,83 +39,67 @@ func NewShort(
 	if err != nil {
 		panic(err)
 	}
-	return []router.Route{
-		{
-			Method: "GET",
-			Path:   "/oauth/github/sign-in",
-			Handle: NewSSOSignIn(
-				sso.SingleSignOn(githubSSO),
-				webFrontendURL,
-			),
-		},
-		{
-			Method: "GET",
-			Path:   "/oauth/github/sign-in/callback",
-			Handle: NewSSOSignInCallback(
-				sso.SingleSignOn(githubSSO),
-				*frontendURL,
-			),
-		},
-		{
-			Method: "GET",
-			Path:   "/oauth/facebook/sign-in",
-			Handle: NewSSOSignIn(
-				sso.SingleSignOn(facebookSSO),
-				webFrontendURL,
-			),
-		},
-		{
-			Method: "GET",
-			Path:   "/oauth/facebook/sign-in/callback",
-			Handle: NewSSOSignInCallback(
-				sso.SingleSignOn(facebookSSO),
-				*frontendURL,
-			),
-		},
-		{
-			Method: "GET",
-			Path:   "/oauth/google/sign-in",
-			Handle: NewSSOSignIn(
-				sso.SingleSignOn(googleSSO),
-				webFrontendURL,
-			),
-		},
-		{
-			Method: "GET",
-			Path:   "/oauth/google/sign-in/callback",
-			Handle: NewSSOSignInCallback(
-				sso.SingleSignOn(googleSSO),
-				*frontendURL,
-			),
-		},
-		{
-			Method: "GET",
-			Path:   "/r/:alias",
-			Handle: NewLongLink(
-				instrumentationFactory,
-				shortLinkRetriever,
-				timer,
-				*frontendURL,
-			),
-		},
-		{
-			Method: "GET",
-			Path:   "/features/:featureID",
-			Handle: FeatureHandle(
-				instrumentationFactory,
-				featureDecisionMakerFactory,
-				authenticator,
-			),
-		},
-		{
-			Method: "GET",
-			Path:   "/analytics/track/:event",
-			Handle: analytics.TrackHandle(instrumentationFactory),
-		},
-		{
-			Method: "POST",
-			Path:   "/api/search",
-			Handle: SearchHandle(search),
-		},
-	}
+
+	var shortRouter shortRouter
+
+	shortRouter.addRoute("GET", "/oauth/github/sign-in", handle.NewSSOSignIn(
+		sso.SingleSignOn(githubSSO),
+		webFrontendURL,
+	))
```

Comment:
It does look more concise. However, we can't see the parameter name any more.

Reply:
I will revert this part